### ISSUE TITLE
Add an option to block incoming email that is considered spam

### ISSUE THREAD

**Issue by scanner** [scanner/as_email_service#111]
Sometimes we want some accounts do not get spam email delivered to them at all.

This is especially relevant for email accounts that have forwarding set and the mail service being forwarded to is sending back spam denial messages. This could cause our mail provider to black list some addresses, and in those cases it is best to not forward the message at all.

This is already happening with at least one account we forward email to so we need this filtering in place before we start forwarding email for that account for real.

**Comment by scanner** [comment]
Duplicate of GH-43
